requirements/cache.txt

```diff
@@ -0,0 +1 @@
+a
```

Comment:
Is this right? Just double checking since it looked a little funny.

Reply:
That was me. There was a case where the action cache was essentially "stuck". The cache key had been created without storing the image, and so it couldn't be deleted, recalled, or overwritten. I created this file to change the hash key.